framework/src/modules/interoperability/base_cc_command.ts

```diff
@@ -16,16 +16,42 @@ import { Schema, emptySchema } from '@liskhq/lisk-codec';
 import { NamedRegistry } from '../named_registry';
 import { CCCommandExecuteContext, ImmutableCrossChainMessageContext } from './types';
 
+/**
```

Comment:
The generated documentation for `BaseCCCommand`'s execute and verify methods is somehow not able to link to the `CCCommandExecuteContext<T>` type.

Reply:
exported it now